pkg/controller/admin/events.go

```diff
@@ -74,11 +74,9 @@ func (c *Controller) HandleEventsShow() http.Handler {
 				Name:  "System",
 			}
 		default:
-			if id, err := strconv.ParseUint(realmID, 10, 64); err != nil {
-				scopes = append(scopes, database.WithAuditRealmID(uint(id)))
-			}
-
-			if realm, err = c.db.FindRealm(realmID); err != nil {
+			scopes = append(scopes, database.WithAuditRealmID(realmID))
```

Comment:
```
realm, err = c.db.FindRealm(realmID)
scopes = append(scopes, database.WithAuditRealmID(realm.ID))
```

I think this would be the same without needing interface{} params necessarily

Reply:
Yea, but that adds another db call and I think it leaks the existence of a realm